manim/mobject/three_dimensions.py

```diff
@@ -732,6 +791,71 @@ def get_start(self):
     def get_end(self):
         return self.end
 
+    @staticmethod
+    def perp_to_line(point, line, **kwargs):
+        """Returns a line passing through a point and
+        perpendicular to another line.
+
+        Example
+        -------
+
+        .. manim:: PerpendicularLineExample
+            :save_last_frame:
+
+            class PerpendicularLineExample(ThreeDScene):
+                def construct(self):
+                    self.set_camera_orientation(45 * DEGREES, 135 * DEGREES)
+                    ax = ThreeDAxes()
+                    line1 = Line3D([-5, -5, -5], [5, 5, 5], color=RED)
+                    dot = Dot3D([3, 0, 2], color=BLUE)
+                    line2 = Line3D.perp_to_line(dot.get_center(), line1, color=YELLOW)
+                    self.add(ax, line1, dot, line2)
+
+        Parameters
+        ----------
+        point: :class:`numpy.ndarray`
+            A numpy array of the point at which the line passes
+        line: :class:`Line3D`
+            The line to be perpendicular to.
+        """
+        u = point - line.get_center()
+        v = line.vect
+        point2 = projected_vector(v, u)
+        direction = point - point2
+        return Line3D(point2 + direction * 10, point2 - direction * 10, **kwargs)
```

Comment:
It'd probably make more sense if the generated line was the same length as the line that it's based on.